docs/site/src/demos/chips/Chips.js

```diff
@@ -0,0 +1,65 @@
+// @flow weak
+
+import React, { PropTypes } from 'react';
+import { createStyleSheet } from 'jss-theme-reactor';
+import Avatar from 'material-ui/Avatar';
+import Chip from 'material-ui/Chip';
+import Face from 'material-ui/svg-icons/face';
+import { grey } from 'material-ui/styles/colors';
+import avatarImage from 'docs/site/assets/images/uxceo-128.jpg';
+
+const styleSheet = createStyleSheet('Chips', () => ({
+  chip: {
+    margin: '0 8px',
+  },
+  svgIcon: {
+    color: grey[800],
+  },
+  row: {
+    display: 'flex',
+    justifyContent: 'center',
+  },
+}));
+
+function handleRequestDelete() {
+  alert('You clicked the delete icon.');
+}
+
+function handleClick() {
+  alert('You clicked the Chip.');
+}
+
+export default function Chips(props, context) {
+  const classes = context.styleManager.render(styleSheet);
+  return (
+    <div className={classes.row}>
+      <Chip
```

Comment:
This element is focusable but don't have `:focus` state.
Is that expected?
![capture d ecran 2017-01-10 a 23 53 46](https://cloud.githubusercontent.com/assets/3165635/21828303/101d16b2-d790-11e6-9e51-88a1275c9609.png)